fuzzy/docker/apps/fsl/neurodebian-travis.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Could we move this file to `fuzzy/docker/resources/`? I think it may be useful for a lot of libraries, not just FSL, so bringing it out there may make sense.